app/main.py

```diff
@@ -164,5 +164,21 @@ def datasets_by_project_id(project_id):
     else:
         abort(404, description="Resource not found")
 
-
-
+@app.route("/get_email/<int:owner_id>", methods=["GET"])
+def get_owner_email(owner_id):
+    bf = Blackfynn(
+        api_token=Config.BLACKFYNN_API_TOKEN,
+        api_secret=Config.BLACKFYNN_API_SECRET,
+        env_override=False,
+        host=Config.BLACKFYNN_API_HOST
+    )
+    
+    # Filter to find user based on provided int id
+    org = bf._api._organization
+    members = bf._api.organizations.get_members(org)
+    res = list(filter(lambda x: x.int_id == owner_id, members))
+
+    if not res:
+        abort(404, description="Owner not found")
+    else:
+        return res[0].email
```

Comment:
Could we return a JSON response here? Eg something simple like `{ "email": "nih-data-portal@blackfynn.com"}`. That will allow us to add additional information to the response if we ever want to.

Reply:
Good point